recipe/meta.yaml

```diff
@@ -24,19 +26,19 @@ requirements:
     - perl
   host:
     - python 3.6
-    - cairo
+    - cairo  # [not win]
```

Comment:
The `ft` header relates to Cairo's integration with the FreeType library. We probably need to either (1) add FreeType support to Windows Cairo (I just checked and we *do* package FreeType on Windows) or (2) do something to make poppler not require Cairo's FreeType support on Windows.

Reply:
I think we might have to do (1) - I couldn't see any way to stop it wanting `cairo-ft.h`. I've had to disable the `pdftocairo` check on Windows, but I think this is otherwise harmless.